manticore/core/cpu/abstractcpu.py

```diff
@@ -363,17 +369,15 @@ class Cpu(Eventful):
 
     def __init__(self, regfile, memory, **kwargs):
         assert isinstance(regfile, RegisterFile)
+        disasm = kwargs.pop("disasm", None)
```

Comment:
This is in place due to the additional argument from the constructor, will be used to initialize the disassembler similarly to how the disasm is set in line 380. I was planning to push a commit in a new PR with the plain binja disassembler (not IL) using this.